activities/Moon.activity/css/activity.css

```diff
@@ -77,7 +77,7 @@ canvas {
 }
 
 .tutorial-prev-icon2 {
-	background-image: url(../icons/go-left.svg);
+	background-image: url(../../../icons/go-left.svg);
```

Comment:
Not at all a good idea to rely on icons in Sugarizer Core. Each activity should be deployable by itself